Updated reCAPTCHA with PHP config note (#5998)
magento/devdocs: guides/v2.3/security/google-recaptcha.md

## Configure reCAPTCHA

{:.bs-callout .bs-callout-warning}

> Before reCAPTCHA can be configured, you must ensure that your `PHP.ini` file includes the following setting: `allow_url_fopen = 1`. See [Required PHP Settings]({{page.baseurl}}/install-gde/prereq/php-settings.html}}).

Reviewer: ```suggestion
Before reCAPTCHA can be configured, ensure that your `PHP.ini` file includes this setting: `allow_url_fopen = 1`. See [Required PHP Settings]({{page.baseurl}}/install-gde/prereq/php-settings.html}}).
```
